### PR TITLE
you can no longer raise the living into ghouls

### PR DESCRIPTION
**Pull request by tralezab**
## About The Pull Request

bad mistake in heretic code caught

## Why It's Good For The Game

you can't believe how fuckin' annoying this is

## Changelog
:cl:
fix: heretics raising the living into ghouls
/:cl:

### CODE REVIEW

File: code/modules/antagonists/eldritch_cult/knowledge/flesh_lore.dm [hunk 1]
```diff
@@ -25,6 +25,8 @@
 	var/mob/living/carbon/human/humie = locate() in atoms
 	if(!humie)
 		return
+	if(humie.stat != DEAD)
+		return
 
 	humie?.grab_ghost()
```
Comment (tralezab) [inline code/modules/antagonists/eldritch_cult/knowledge/flesh_lore.dm:31]: also i don't wanna be that guy but now just looking at this do we even need the ? if we return early on !humie

**Comment by Angustmeta** [comment]
You can even click yourself with mansus grasp to turn yourself into a ghoul slaved to yourself

**Comment by tralezab** [comment]
oh my god

**Comment by tralezab** [comment]
who writes this with a straight face bro...

File: code/modules/antagonists/eldritch_cult/knowledge/flesh_lore.dm (hunk 1 above)
Comment (bobbahbrown) [reply code/modules/antagonists/eldritch_cult/knowledge/flesh_lore.dm:31]: you are correct

Comment (tmtmtl30) [reply code/modules/antagonists/eldritch_cult/knowledge/flesh_lore.dm:31]: you could get rid of the ? but not the if(!humie) check

**Comment by ExcessiveUseOfCobblestone** [comment]
line 57 is undocumented cant believe someone let this get merged smh!

**Comment by ATH1909** [comment]
holy shit this is amazing

**Comment by ATH1909** [comment]
add this check to flesh!mansus grasp as well

**Comment by ATH1909** [comment]
also also, could you make the flesh!mansus grasp mark detonation effect not happen if you successfully raise your target as a ghoul?

**Comment by tralezab** [comment]
i tried testing this and another ritual revived myself as the heretic into some other mob summon, WHILE ALIVE, AS THE INVOKER

HEEEEEEEEEELP

**Comment by EdgeLordExe** [comment]
how the fuck. selected atoms prohibits anyone that id living from being slected as a source. what the actual fuck. i eill have to investigate this

**Comment by EdgeLordExe** [comment]
and also good catch athath i wilk make a cleanup pr tommorow morning

**Comment by ATH1909** [comment]
note that I haven't actually tested those claims about flesh!mansus grasp, but it looks like the code for flesh!mansus grasp has the same issue as the code for voiceless dead

**Comment by EdgeLordExe** [comment]
what the fuck i clearly remember putting a dead check for these. god

**Review by ShizCalev — APPROVED** [review]

**Pull request merged**